appyters/GSEA_Appyter/GSEA_Appyter.ipynb

```diff
@@ -117,7 +119,7 @@
     "                default='GSE70466.gct',\n",
     "                examples={\n",
     "        'GSE70466.gct': 'https://appyters.maayanlab.cloud/storage/GSEA_Appyter/GSE70466.gct'},\n",
-    "                required=True,\n",
+    "                required='true',\n",
```

Comment:
What's the reason for this change? Does `True` not work? In that case this looks like a bug in appyter.. "true" as a string seems like a hack

Reply:
Yes, this was something I was trying out, but neither `True` nor `'true'` so I ended up not using it, but I forgot to remove it. I think it may be because it is within a ChoiceField? I will remove it.

Reply:
@cindyxio I found the issue with required so if you leave it as True then it will work in the newer version of appyters when I release it